packages/medusa/src/api-v2/admin/fulfillments/[id]/route.ts

```diff
@@ -0,0 +1,38 @@
+import { cancelFulfillmentsWorkflow } from "@medusajs/core-flows"
+import { ModuleRegistrationName } from "@medusajs/modules-sdk"
+import {
+  AdminFulfillmentDeleteResponse,
+  IFulfillmentModuleService,
+} from "@medusajs/types"
+
+import {
+  AuthenticatedMedusaRequest,
+  MedusaResponse,
+} from "../../../../types/routing"
+
+export const DELETE = async (
```

Comment:
@olivermrbl since we're not exposing delete for fulfillment in the module, am i right in assuming that this is a cancel endpoint? If so, wouldn't it make more sense to call this `POST /admin/fulfillments/:id/cancel`?

Reply:
I think that makes sense, since we are not deleting anything here.

Reply:
Yup, makes sense 👍